docs/Configuration.md

```diff
@@ -17,8 +17,10 @@ browser_params = [BrowserParams() for _ in range(manager_params.num_browsers)]
 where `manager_params` is of type `class<ManagerParams>` and `browser_params` is a length 5 list
 of configurations of `class<BrowserParams>`.
 
-####Validations:
+## Validations:
```

Comment:
We could move the`Validations` section below, perhaps after `Browser Configuration Options`, as it looks weird being above the index

Reply:
I moved it under browser and manager_params because it felt weird to have it be the first thing in there.

Reply:
> However now it's breaking up two parts that used to fit together quite nicely.

I see, I kind of noticed that too. Alternatively, we could keep `Validations` where it was, in bold but not as a heading.

Reply:
Let's keep it in it's new position for now and rethink this once we rewrite it in .rst